web/src/components/KeypointLabel/KeypointLabel.jsx

```diff
@@ -0,0 +1,66 @@
+import { keypointsText, keypointsIndex } from "../../utils/constant";
+import styles from "./KeypointLabel.module.css";
+import clsx from "clsx";
+import tickGreenIcon from "../../assets/tick-green-icon.svg";
+import eraserIcon from "../../assets/cards_school_eraser.svg";
+export function KeypointLabel({
+  selected,
+  marked,
+  keypoints = keypointsIndex,
+  onKeypoint,
+  onComplete,
+  isRemoveKeypoint, 
+  setIsRemoveKeypoint
+}) {
+  return (
+    <div className={styles.keypointContainer}>
+      {keypoints.map((value, index) => {
+        const isMarked = marked.includes(value);
+        return (
+          <div className={styles.keypointOption} key={index}>
+            <button
+              type="button"
+              className={clsx(
+                styles.keypointButton,
+                isMarked
+                  ? styles.marked
+                  : value === selected
+                  ? styles.selected
+                  : ""
+              )}
+              onClick={() => onKeypoint(value)}
```

Comment:
```suggestion
              onClick={() => onKeypoint(value)&setIsRemoveKeypoint(false)}
```
Disable eraser when selecting a keypoint, hopefully this can improve user experience.